deployment/docker/alerts.yml

```diff
@@ -125,7 +125,7 @@ groups:
             When VictoriaMetrics constantly hits the limit it means that storage is overloaded and requires more CPU."
 
       - alert: TooManyLogs
-        expr: sum(increase(vm_log_messages_total{level!="info"}[5m])) by (job, instance) > 0
+        expr: sum(increase(vm_log_messages_total{level!="error"}[5m])) by (job, instance) > 0
```

Comment:
it should be `level!="error"`
otherwise, it rules out all `error` messages and will alert only on `info` and `warning`